src/components/radio-button/radio-button.spec.tsx

```diff
@@ -46,48 +49,44 @@ const borderColorsByValidationTypes = {
 describe("RadioButton", () => {
   describe("propTypes", () => {
     it("does not allow a children prop", () => {
-      const consoleSpy = jest.spyOn(console, "error");
-      render({ children: "someChildren" });
+      const consoleSpy = jest
+        .spyOn(console, "error")
+        .mockImplementation(() => {});
```

Comment:
You left this mock just to not pollute the logs during running the unit tests?

Reply:
yes, that's the exact reason for it to be here